lib/fluent/config/literal_parser.rb

```diff
@@ -147,7 +147,7 @@ def scan_embedded_code
 
         @ss.pos += code.length
 
-        code
+        '"#{' + code + '}"'
```

Comment:
In previous code, we get `code` as `ENV['HOGE']` for `"#{ENV['HOGE']}foo"`, which is okay.
But, we get `code` as `ENV['HOGE']}foo#{ENV['MOGE']` for `"#{ENV['HOGE']}foo#{ENV['MOGE']}"`, whish is bad.